src/utils/composeStores.js

```diff
@@ -1,11 +1,18 @@
-import mapValues from '../utils/mapValues';
-import pick from '../utils/pick';
+import mapValues from './mapValues';
+import pick from './pick';
+
+// UNDEF = undefined;
+const UNDEF = void 0;
```

Comment:
You can replace `return UNDEF` with just `return`, and use `typeof x === 'undefined'` as a check.
Or are you referring to something else?